modules/apps/object/object-test/src/testFunctional/tests/ObjectsPopulating.testcase

```diff
@@ -483,12 +483,62 @@ definition {
 	}
 
 	@description = "LPS-137865 - Verify that it is possible to submit an entry with a Double field blank that is not required"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanSubmitEntryWithDoubleFieldBlank {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "String",
+			fieldName = "stringField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Double",
+			fieldName = "doubleField",
+			fieldType = "Double",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		PopulateObjectFormsConfiguration.openForm();
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
 
-		// TODO LPS-141823 CanSubmitEntryWithDoubleFieldBlank pending implementation
+		PopulateObjectFormsConfiguration.openFormSettings();
+
+		PopulateObjectFormsConfiguration.selectObject(label = "Object");
+
+		PopulateObjectFormsConfiguration.selectCustomObject(label = "Custom Object");
+
+		Form.gotoAddField(fieldType = "Text");
+
+		PopulateObjectFormsConfiguration.mapFormField(fieldLabelName = "String");
```

Comment:
Hi @andreefarias 

You need to add the double field for the form too and map it.